moto/moto_proxy/proxy3.py

```diff
@@ -62,15 +62,15 @@ def parse_request(
         host: str,
         path: str,
         headers: Any,
-        body: bytes,
+        body: str,
```

Comment:
This is not correct - the body of a request could be either bytes (in case of an image upload, for instance) or a string. 

What was the reason behind this change?

Reply:
Looks like it was a very incorrect type from botocore-stubs on the arguments to AWSPreparedRequest. Fixed.

Looking at the implementation, I think the true type of the body argument to AWSPreparedRequest is something like str | bytes | bytearray | IOBase | None. They just have `str` right now. I'll take it up with them.